development-docs/TESTING.md

```diff
@@ -144,125 +145,124 @@ You need to use the `groups` system property in order to execute a group of syst
 
-`KUBERNETES_DOMAIN` should be specified only in case you are using specific configuration in your kubernetes cluster.
+`KUBERNETES_DOMAIN` should be specified only in case, when you are using specific configuration in your kubernetes cluster.
 
 ##### Specific Kafka version
 
-To set custom Kafka version in system tests you need to set environment variable `ST_KAFKA_VERSION` to one of the values in [kafka-versions](kafka-versions.yaml).
+To set custom Kafka version in system tests you need to set the environment variable `ST_KAFKA_VERSION` to one of the values in [kafka-versions](kafka-versions.yaml).
 
 ##### Cluster Operator Log level
 
 To set the log level of Strimzi for system tests you need to set the environment variable `STRIMZI_DEFAULT_LOG_LEVEL` with one of the following values: `ERROR`, `WARNING`, `INFO`, `DEBUG`, `TRACE`.
 
 ## Use Remote Cluster
 
-The integration and system tests are run against a cluster specified in the environment variable `TEST_CLUSTER_CONTEXT`. 
-If this variable is not set, the kubernetes client will use the currently active context. 
+The integration and system tests are ran against a cluster specified in the environment variable `TEST_CLUSTER_CONTEXT`.
+If this variable is unset, the kubernetes client will use the currently active context.
 Otherwise, it will use the context from kubeconfig with a name specified by the `TEST_CLUSTER_CONTEXT` variable.
 
-For example command `TEST_CLUSTER_CONTEXT=remote-cluster ./systemtest/scripts/run_tests.sh` will execute tests with cluster context `remote-cluster`. 
+For example, command `TEST_CLUSTER_CONTEXT=remote-cluster ./systemtest/scripts/run_tests.sh` will execute tests with cluster context `remote-cluster`.
 However, since system tests use command line `Executor` for some actions, make sure that you are using context from `TEST_CLUSTER_CONTEXT`.
 
-System tests uses admin user for some actions. 
+System tests uses admin user for some actions.
 You can specify the admin user using variable `TEST_CLUSTER_ADMIN` (by default it uses `developer` because `system:admin` cannot be used over remote connections).
 
 ## Helper script
 
-The `./systemtest/scripts/run_tests.sh` script can be used to run the `systemtests` using the same configuration as used in the travis build. 
+The `./systemtest/scripts/run_tests.sh` script can be used to run the `systemtests` using the same configuration as used in the travis build.
 You can use this script to easily run the `systemtests` project.
 
 Pass additional parameters to `mvn` by populating the `EXTRA_ARGS` env var.
 
     EXTRA_ARGS="-Dfoo=bar" ./systemtest/scripts/run_tests.sh
-    
+
 ## Running single test class
 
-Use the `verify` build goal and provide a `-Dit.test=TestClassName[#testMethodName]` system property. 
+Use the `verify` build goal and provide `-Dit.test=TestClassName[#testMethodName]` system property.
 
     mvn verify -pl systemtest -Dit.test=KafkaST#testKafkaAndZookeeperScaleUpScaleDown
```

Comment:
Can you please change the `testKafkaAndZookeeperScaleUpScaleDown` to `testCustomAndUpdatedValues`? This test is no longer part of the `KafkaST` and if someone will try it, he will see some errors and maybe he will think that he is doing something wrong. Thanks :smile:

Reply:
Done.